### PR TITLE
Hz search

### PR DESCRIPTION
**Pull request by hannah**

### CODE REVIEW

**Comment by hannah** [comment]
@HeroicEric  please review

**Comment by HeroicEric** [comment]
@hzab Looks like you forgot to remove a `binding.pry` from the sign_in helper and it's causing the build to fail

File: app/controllers/mugshots_controller.rb [hunk 1]
```diff
@@ -22,8 +22,12 @@ def show
   end
 
   def index
-    @mugshots = Mugshot.order(:created_at).page params[:page]
-
+    if params[:q]
+      first_name = params[:q][:first_name]
+      @mugshots = Mugshot.where("first_name ILIKE ?", "%#{first_name}%").page params[:page]
```
Comment (HeroicEric) [inline app/controllers/mugshots_controller.rb:27]: Should we also order these results?

File: spec/support/user_signs_in_helper.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 module UserSignsInHelper
   def sign_in_as(user)
+    binding.pry
```
Comment (HeroicEric) [inline spec/support/user_signs_in_helper.rb:3]: This should be removed.

**Comment by hannah** [comment]
whoops, I'll fix those in a bit

**Pull request closed without merging**